capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -14,15 +14,9 @@
 
 package com.google.step.clients;
 
-import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Query;
-import com.google.appengine.api.datastore.PreparedQuery;
-import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.*;
```

Comment:
Try to avoid just importing everything, and import only the things you will actually use.

Reply:
Here's a quote from when Martin encountered the same issue: "If you go into File > Settings, under Editor > Code Style > Java, in the Imports tab, you can change the value of "Class count to use import with '*'" and "Names count ..." to some really large value (I used 100 for each) since you can't change it to be unlimited."

Reply:
Thanks for the tip guys, I imported everything individually again!